pkg/skaffold/runner/runner_test.go

```diff
@@ -379,14 +378,12 @@ func TestNewForConfig(t *testing.T) {
 					DeployType: latest.DeployType{
 						KubectlDeploy:   &latest.KubectlDeploy{},
 						KustomizeDeploy: &latest.KustomizeDeploy{},
-						HelmDeploy:      &latest.HelmDeploy{},
```

Comment:
Hmm, this test checks for the ordering of the deployers and should be retained.

Reply:
Thanks for the find.  I will fix it.